### ISSUE TITLE
Media oembed page missing head html

### ISSUE THREAD

**Issue by tommyneu** [unlcms/project-herbie#932]
The media oembed page is missing essential html causing accessibility issues. We at least need to define the title tag and the favicon for those pages.

Example link: 
https://veteranstribute.unl.edu/media/oembed/?url=https%3A//mediahub.unl.edu/media/19835&max_width=0&max_height=0&hash=uO_6bE0Jns3py6lMzxcgqIqQWPEuSgEdq7dcv3_z9lw

**Comment by sarongebre** [comment]
@tommyneu, where do you see this link being used? I don't think this is a page generated by a. site admin. It's a link used in the inner HTML of a media oEmbed file... It's not supposed to be used like that on its own...

**Comment by tommyneu** [comment]
When you embed a mediahub video, the page gets a iframe with that oembed url which itself has an iframe with the actual mediahub embed in it.

That intermediate oembed page is missing things like the title tag and other things which cause it to fail accessibility tests. That page is auto generate and not typically used by itself but it still needs to pass accessibility tests.
